### ISSUE TITLE
2 people on the Hub the colors mix

### ISSUE THREAD

**Issue by teguria** [IBVSoftware/CollaBoard-Feedback#378]
v 1.118
Hub
Key: 2760
Code: uzqqu

2 people on the Hub using the QAT.
the 2 were inking at the same time in different places of the canvas.
Probably when they tap at the same time and release

**Comment by corradocavalli** [comment]
@teguria Try, at least, to try to see if it is possible to provide reproducible scenario to identify and fix this kind of issue, the effort required on your side is exactly the same we have to spent to try to figure out what's happening and we don't have enough time and resource to do both discovering and fixing.

**Comment by gsantopaolo** [comment]
@teguria do you remeber i showed you the ink colors issue by design we have?
maybe is this case?

**Comment by teguria** [comment]
yes I think it's the case @gsantopaolo but not 100% sure. Will test it when we have the new release
